File: Keep-Note-Complete/keepnote_step6/NoteService/NoteService.API/Models/Reminder.cs
using System;
using Newtonsoft.Json;
using Newtonsoft.Json.Serialization;
using MongoDB.Bson.Serialization.Attributes;

namespace NoteService.API.Models
{
    public class Reminder
    {
        [BsonId]
        [JsonProperty(PropertyName = "id")]
        public int Id { get; set; }
        [JsonProperty(PropertyName = "description")]
        public string Description { get; set; }
        [JsonProperty(PropertyName = "name")]
        public string Name { get; set; }
        [JsonProperty(PropertyName = "type")]
        public string Type { get; set; }
        [JsonProperty(PropertyName = "createdby")]
        public string CreatedBy { get; set; }
        [JsonProperty(PropertyName = "createddate")]
        public DateTime CreationDate { get; set; }
    }
}

File: Keep-Note-Complete/angular-final-assignment/src/app/Reminder.ts
        export class Reminder {
            id: Number;
            name: string;
            description:string;    
            createdBy:string;
            type:string;
            createdDate:Date;
            
            constructor() {
              this.name = '';
              this.description = '';
              this.type ='';
            }
         }
          